drivers/src/lib.rs

```diff
@@ -215,7 +215,7 @@ mod tests {
     #[test]
     fn build_cpu_info() {
         let tree = stub_tree();
-        CpuInfo::parse_from(&tree);
+        let _ = CpuInfo::parse_from(&tree);
```

Comment:
Don't we want to panic on error here from the Result? (for the unit test)?

Reply:
Yes of course, don't know why I did that... I'll simply unwrap.